spring-mobile/template.mf

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: org.togglz.spring.mobile
+Bundle-Vendor: http://www.togglz.org/
+Bundle-Version: ${project.version}
+Bundle-Name: ${project.name}
+Version-Patterns:
+ default;pattern="[=.=.=, =.+1.0)"
+Import-Template:
```

Comment:
Looks fine to me. As I wrote before: I'm not so familiar with OSGi myself, but the Maven plugin will fail if there are any packages missing. So if the build is green everything is fine.